gui/src/installer/view/editor/mod.rs

```diff
@@ -271,11 +271,14 @@ pub fn edit_key_modal<'a>(
             Column::new()
                 .spacing(25)
                 .push(Row::new()
+                    .push(Space::with_width(10))
```

Comment:
The title and the "Select the signing device..." wording below previously had the same left-alignment. Would it be possible to remove this space in order to keep them aligned?